2023/speakers/DIPanelists.html

```diff
@@ -0,0 +1,139 @@
+                <img src="https://nescalas.github.io/2020/images/nescalas-logo.png" style="height: auto; max-width: 75px">
+              </a>NEScala
+            </div>
+          </div>
+          <div class="collapse navbar-collapse" id="main-navbar">
+            <ul class="navbar-nav mr-auto w-100 justify-content-end">
+              <li class="nav-item">
+                <a class="nav-link" href="https://nescalas.github.io">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#speaker">Panelists</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Header End -->
+
+    <!-- Hero Section Start -->
+  <section id="main">
+    <div class="text-center title-area">
+        <div class="d-flex justify-content-center align-items-center h-100 text-white">
+          <h1 class="display-3">Northeast Scala Symposium<br>2023</h1>
+        </div>
+    </div>
+  </section>
+  <!-- Hero Section End -->
+    
+    <!-- Panelist Start -->
+    <section id="speaker" class="section-padding speaker">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-3 col-lg-3 col-xs-12">
+            <div class="speaker-photo">
+              <img class="img-fluid" src="https://nescalas.github.io/assets/img/Jasna.jpg" alt="">
+            </div>
+          </div>
+          <div class="col-md-9 col-lg-9 col-xs-12 clearfix">
+            <h2>Jasna Rodulfa-Blemberg</h2>
+            <p>I am a software engineer on a growing Machine Learning Orchestration team at Etsy, where we work with the Twitter stack and, increasingly, the Cats + Cats Effect stack in distributed systems. I am self-taught, having majored in the humanities and stumbling into software engineering when a recruiter gave me a chance from seeing coding as my hobby. In my professional career I started developing in PHP and JavaScript, moved to full stack with Java, and since 2017, have transitioned full-time to the backend with Scala. In my free time, I love playing video games and board games, reading and watching sci-fi stories, traveling, staying fit, cooking for loved ones, and trying different restaurants, I also joined the Typelevel Steering Committee in 2022 and am NEScala's 2023 DEIB Safety Team lead. I am based in the Washington, DC, USA area and live happily with my spouse.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section-padding speaker">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-3 col-lg-3 col-xs-12">
+            <div class="speaker-photo">
+              <img class="img-fluid" src="https://nescalas.github.io/assets/img/AdamEdwards.jpg" alt="">
+            </div>
+          </div>
+          <div class="col-md-9 col-lg-9 col-xs-12 clearfix">
+            <h2>Adam "Decorum" Edwards</h2>
+            <p>No bio.</p>
```

Comment:
```suggestions
<p>I'm a Sr. Software Engineer at Disney Streaming Services. I first used Scala at Bank of America in 2014 and reconnected with it in 2018 when I joined the team at Rally Health, and I have been a big fan ever since! While I may not be super-involved in the Scala community, I really enjoy the language and love to encourage new and aspiring developers to explore the language. Excited to share and explore ways that we as developers, and members of hiring teams, can strive to make the community more inclusive!</p>
```